Update ADA rosinstalls.

egordon opened the pull request:
Switch out old ADA rosinstalls for one that works with current feeding demo.

brianhou commented:
I suggested deleting all the old rosinstalls and only keeping the new ones that reflect our current stack. I think it would be a good idea to have all the ones that you suggest, but those could perhaps be resolved in future PRs?

egordon commented:
@gilwoolee @brianhou I just added an ada-feeding-sim rosinstall, since that was easy to make, but I agree that it would be good to unblock this PR and look to future PRs to add other, more minimal rosinstalls. I added place-holder issue #54

brianhou commented:
Is there some order that the repositories are listed in? If not, it'd be helpful to do something like alphabetical so it's easier to pick out differences.

gilwoolee commented on ada-feeding-sim.rosinstall:
```diff
@@ -0,0 +1,11 @@
+- git: {local-name: pr_control_msgs, uri: 'https://github.com/personalrobotics/pr_control_msgs.git'}
+- git: {local-name: pr_ros_controllers, uri: 'https://github.com/personalrobotics/pr_ros_controllers.git'}
+- git: {local-name: pr_tsr, uri: 'https://github.com/personalrobotics/pr_tsr.git'}
+- git: {local-name: rewd_controllers, uri: 'https://github.com/personalrobotics/rewd_controllers.git'}
```
Need to add the following repositories:
`bite_selection_package`
`food_detector`
`pytorch_retinanet`
(These are no longer submodules of food_detector)
I also want to add `sim_face_detector` to `face_detection` repository and make that a sim-dependency as well.
I can work on that tomorrow morning and let you know when it's ready. 

Why do we need rewd_controllers, pr_ros_controllers, and pr_control_msgs here?

gilwoolee commented on ada-feeding.rosinstall:
```diff
@@ -0,0 +1,17 @@
+- git: {local-name: pr_hardware_interfaces, uri: 'https://github.com/personalrobotics/pr_hardware_interfaces.git'}
+- git: {local-name: pr_ros_controllers, uri: 'https://github.com/personalrobotics/pr_ros_controllers.git'}
+- git: {local-name: pr_tsr, uri: 'https://github.com/personalrobotics/pr_tsr.git'}
+- git: {local-name: rewd_controllers, uri: 'https://github.com/personalrobotics/rewd_controllers.git'}
```
As mentioned above, we need `bite_selection_package` and `pytorch_retinanet` as well.

gilwoolee commented:
Looks mostly good except for the requested changes, but I'd like to wait until we test this on a clean workspace, for both sim and real.
